File: 1001/1001.py
"""
***@Time    : 2019-10-22
***@Author  : hushengquan
***@File    : 1001.py
***@Software: vim
***@type    : pat practice
"""
num = int(input())
count = 0

while (num != 1):
    if (num % 2 == 0):
        num = num / 2
    else:
        num = (num * 3 + 1) / 2
    count += 1

print(count)

File: 1001/1001.cpp
/***********************************************
 > @Time       : 2019-10-29
 > @Author     : hushengquan
 > @File       : 1001.cpp
 > @Software   : vim
 > @type       : pat practice
************************************************/
#include <iostream>
using namespace std;

int main() {
    int n, count = 0;
    cin >> n;

    while (n != 1) {
        if (n % 2 != 0) {
            n = 3 * n + 1;
        }
        n /= 2;
        count++;
    }

    cout << count;
    return 0;
}
